specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-06-01/machineLearningServices.json

```diff
@@ -0,0 +1,4054 @@
+              "$ref": "#/definitions/ListWorkspaceQuotas"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List workspace quotas by VMFamily": {
+            "$ref": "./examples/ListWorkspaceQuotasByVMFamily.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/workspaces": {
+      "get": {
+        "tags": [
+          "Workspaces"
+        ],
+        "description": "Lists all the available machine learning workspaces under the specified subscription.",
+        "operationId": "Workspaces_ListBySubscription",
+        "x-ms-examples": {
+          "Get Workspaces by subscription": {
+            "$ref": "./examples/getWorkspacesBySubscription.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PaginationParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/WorkspaceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/connections": {
+      "get": {
+        "tags": [
+          "WorkspaceConnections"
+        ],
+        "x-ms-examples": {
+          "ListWorkspaceConnections": {
+            "$ref": "./examples/listWorkspaceConnections.json"
+          }
+        },
+        "operationId": "WorkspaceConnections_List",
+        "description": "List all connections under a AML workspace.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/TargetParameter"
+          },
+          {
+            "$ref": "#/parameters/CategoryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the connections.",
+            "schema": {
+              "type": "array",
```

Comment:
Collection GETs should conform to the standard paginated response definition and be marek x-ms-pageable.  See other collection GETs in this spec (i..e "PaginatedComputeResourcesList")

Reply:
We have a restriction in our service to allow only 100 connections to be created for one workspace. So it should be okay to not enable pagination for this API?

Reply:
The response needs to look like a paginated response to conform to the RPC (i.e.  { "value": [] } ) even if it is not paginated (see the other collection responses in this spec).  It doesn't hurt to put "x-ms-pageable": true with "nextLinkName": null as well.  This will make it a non-breaking change to enable pagination in the future if you need to.

Reply:
Okay, got it. I will fix it